.github/workflows/tag_release.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v2
     - name: Install Go
-      uses: actions/setup-go@v2.1.1
+      uses: actions/setup-go@v2.1.3
```

Comment:
if we remove the .1.3 and make it just actions/setup-go@v2 then we get the minors for free